src/network/bufferedconnection.cc

```diff
@@ -304,6 +326,23 @@ void BufferedConnection::start_receiving() {
 	   });
 }
 
+void reduce_send_buffer(boost::asio::ip::tcp::socket& socket) {
```

Comment:
`../src/network/bufferedconnection.cc:329:6: warning: no previous prototype for function 'reduce_send_buffer' [-Wmissing-prototypes]`
I get this warning during compiling.